src/VirtualClient/VirtualClient.Monitors.UnitTests/AtopParserUnitTests.cs

```diff
@@ -629,7 +620,7 @@ public void AtopParserSupportsDefiningExplicitSubsetsOfCounters_Scenario1()
             AtopParser testParser = new AtopParser(rawText);
             IList<Metric> metrics = testParser.Parse();
 
-            Assert.AreEqual(158, metrics.Count);
+            Assert.AreEqual(150, metrics.Count);
```

Comment:
What's the reason for the 8 less metrics? What did we stop parsing?

Reply:
This PR attempts to fix the first record containing a sum of all counters since system boot or counter rollover.  atop does not display any date for several of the counters because during subsequent 1s snapshots the counters remain at 0.  This eliminates records for sdb, sdc in the disk portion (busy, read, write, and avio) 2 x 4 = 8.  It does not affect the inputs for networking for the 60s example provided but potentially any network interface may not be repeated.  Something to watch out for if additional canned input is captured in the future.